packages/gmp.rb

```diff
@@ -22,6 +22,8 @@ class Gmp < Package
      x86_64: '50847bd14c11de841c7c74696e2ff4503253bbaacd38e9ffb31814d6ccfb76f3'
   })
 
+  depends_on 'gcc' # R
```

Comment:
Not a build dependency?

Reply:
Both. `libgmpxx.so` depends on `libstd++.so.6` and `libgcc_s.so.1`.